index.html

```diff
@@ -0,0 +1,103 @@
+                <input type="number"/>
+            </label>
+            <label>
+                Which option best describes your currrent role?
+                <select>
+                    <option>Select current role</option>
+                    <option>Student</option>
+                    <option>Full time job</option>
+                </select>
+            </label>
+            <label>
+                Would you recommend freeCodeCamp to a friend?
+                <div>
+                    <input type="radio" value="Definitely" name="recommend"/> Definitely
+                </div>
+                <div>
+                    <input type="radio" value="Maybe" name="recommend"/> Maybe
+                </div>
+                <div>
+                    <input type="radio" value="Not sure" name="recommend"/> Not Sure
+                </div>
+            </label>
+            <label>
+                What is you favourite feature of freeCodeCamp?
+                <select>
+                    <option>Select an option</option>
+                    <option>Challenges</option>
+                    <option>Projects</option>
+                    <option>Community</option>
+                    <option>Open Source</option>
+                </select>
+            </label>
+            <label>
+                What would you like to see improved? <span>(Check all that apply)</span>
+                <div>
+                    <input type="checkbox" name="impSuggest"/> Front-end Projects
+                </div>
+                <div>
+                    <input type="checkbox" name="impSuggest"/> Front-end Projects
+                </div>
+                <div>
+                    <input type="checkbox" name="impSuggest"/> Front-end Projects
+                </div>
+                <div>
+                    <input type="checkbox" name="impSuggest"/> Front-end Projects
+                </div>
+                <div>
+                    <input type="checkbox" name="impSuggest"/> Front-end Projects
+                </div>
+                <div>
+                    <input type="checkbox" name="impSuggest"/> Front-end Projects
+                </div>
+                <div>
+                    <input type="checkbox" name="impSuggest"/> Front-end Projects
+                </div>
+                <div>
+                    <input type="checkbox" name="impSuggest"/> Front-end Projects
+                </div>
+                <div>
+                    <input type="checkbox" name="impSuggest"/> Front-end Projects
+                </div>
+                <div>
+                    <input type="checkbox" name="impSuggest"/> Front-end Projects
+                </div>
+            </label>
+            <label>
+                Any Comments or Suggestions?
+                <input type="textarea"/>
+            </label>
+            <div class="btn">
+            <button class="button">submit</button>
+        </div>
+        </form>
```

Comment:
A great change you can make to your site to use better semantics. it will not only make your site look cleaner 🌟 but increase your seo ranking 📈 like this:
![semantic tags](https://user-images.githubusercontent.com/125574259/229080922-40a79ed4-7eee-47d1-8222-7d22362bb371.jpeg)